pages/common/git-log.md

```diff
@@ -9,3 +9,11 @@
 - Show the history of a particular file or directory, including differences
 
 `git log -p PATH`
+
+- Show only the first line of each commits
+
+`git log --oneline`
+
+- Quit out of git log
+
+`q`
```

Comment:
This one is kind of generic as it applies to many such commands (man, less, etc.), as described in https://github.com/tldr-pages/tldr/pull/74#issuecomment-35948938. I would suggest not including it.